src/main.rs

```diff
@@ -59,6 +70,112 @@ fn check_valid_env(
     Ok(())
 }
 
+fn current_env() -> HashMap<String, String> {
+    // Create a HashMap from the current set of environment variables (excluding a few)
+    let exclude = ["PS1", "TERM", "HOME"];
+
+    env::vars()
+        .filter(|&(ref k, _)| !exclude.contains(&k.as_str()))
+        .collect()
+}
+
+fn get_ct_vars(
+    org_id: Option<&str>,
+    env: Option<&str>,
+    environments: &Environments,
+) -> Result<HashMap<String, String>> {
+    // Create a HashMap with all the CloudTruth environment values for this environment.
+    let mut ct_vars = HashMap::new();
+    let parameters = Parameters::new();
+    let env_id = environments.get_id(org_id, env)?;
+    let list = parameters.get_parameter_names(org_id, env_id)?;
+    for key in list.iter() {
+        let parameter = parameters.get_body(org_id, env, key)?;
+        // Put the key/value pair into the environment
+        let value = parameter.unwrap_or_else(|| "".to_string());
+        ct_vars.insert(key.to_string(), value);
+    }
+
+    Ok(ct_vars)
+}
+
+fn process_overrides(overrides: Vec<String>) -> HashMap<String, String> {
+    // Create HashMap with all the user-provided overrides
+    let mut over_vars = HashMap::new();
+    for arg_val in overrides {
+        let temp: Vec<&str> = arg_val.splitn(2, '=').collect();
+        if temp.len() != 2 {
+            println!("Ignoring {} due to  no '='", arg_val);
```

Comment:
Leftover?

Reply:
I did want the user to be aware that we had dropped the argument they gave us... Now using the `warn_user()`